src/containers/PlanDetails/PlanDetails.component.js

```diff
@@ -166,6 +168,17 @@ const PlanDetails = ({
             isPartOfCancellationFlow={innerPopup.data.isPartOfCancellationFlow}
           />
         );
+      case 'pauseSubscription':
+        return (
+          <PauseSubscriptionPopup
+            showInnerPopup={showInnerPopup}
+            toOffer={innerPopup.data.offerData}
+            fromOffer={planDetails.offerToSwitch}
+            hideInnerPopup={hideInnerPopup}
+            updateList={updateList}
+            isPartOfCancellationFlow={innerPopup.data.isPartOfCancellationFlow}
+          />
```

Comment:
if we would move it to popupSlice, we could get rid of all of the props

Reply:
That one will be done in the future popup refactor